application/controllers/catalog/Author.php

```diff
@@ -21,6 +21,8 @@ public function index($author_id)
 		$matches = $this->_get_all_author($author_id);
 		$this->data['matches'] = count($matches);
 
+		$this->data['search_order'] = $this->input->get('search_order');
```

Comment:
This gets used to mark the correct sort option as selected in `views/catalog/author.php`